ansible/group_vars/all

```diff
@@ -161,6 +161,8 @@ nginx:
     verify_client: "{{ nginx_ssl_verify_client | default('off') }}"
   wpn:
     router: "{{ nginx_wpn_router | default('1') }}"
+  # an array of user ids to enable extra logging on the system
```

Comment:
an id you mean a namespace?

Reply:
I have added an example to the documentation file